src/style.css

```diff
@@ -0,0 +1,98 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  height: 100%;
+  background: whitesmoke;
+}
+
+ul {
+  list-style-type: none;
+}
+
+#container {
+  display: flex;
+  flex-direction: column;
+  border: 2px solid #000;
+  background-color: #fff;
+  min-width: 450px;
+  width: 40%;
+  position: relative;
+  top: 10%;
+  left: 30%;
+  margin-top: 10%;
+  box-shadow: 15px -2px 63px 13px rgba(0, 0, 0, 0.3);
+}
+
+h1 {
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  font-size: 2rem;
+  padding-left: 10px;
+}
+
+.fa-arrows-rotate {
+  position: absolute;
+  left: 92.5%;
+  bottom: 88%;
+  font-size: 20px;
+  cursor: pointer;
+}
+
+input {
+  border: none;
+  width: 90%;
+  height: 40px;
+  padding-left: 5px;
+}
+
+form input:focus {
+  outline: none;
+  padding-left: 5px;
+}
+
```

Comment:
- [ ] Kindly fix your design to match the [captured video](https://www.youtube.com/watch?v=AcUd-_Yjjqg) and make sure CSS is loaded by *Webpack Style/CSS Loader* 
> Your design
![image](https://user-images.githubusercontent.com/53402600/194138099-6e9c4cdd-7845-4cb9-ac4d-f0730fab8b95.png)